CalibCalorimetry/CaloTPG/src/CaloTPGTranscoderULUT.cc

```diff
@@ -83,6 +83,9 @@ void CaloTPGTranscoderULUT::loadHCALCompress(HcalLutMetadata const& lutMetadata,
       for (unsigned int i = threshold; i < lutsize; ++i)
         if (allLinear_) {
           outputLUT_[index][i] = isOnlyQIE11(id) ? linearQIE11LUT[i] : linearQIE8LUT[i];
+          if (abs(ieta) > 20 && isOnlyQIE11(id) && linearQIE11LUT[i] >= 127) {
```

Comment:
should these numbers be made a name constant? what's 127 exactly? that's the energy? (I understood iete is just and index in eta, but the other one is less obvious)

Reply:
yes, that's a great suggestion. I modified the numbers to a name constant. This number is the saturation value in the 'split cells'